lib/index.js

```diff
@@ -1,5 +1,9 @@
-export default (...args) => (req, res, next) => {
+export default (fallback, ...args) => (req, res, next) => {
   if (req.method === 'GET' && req.accepts('html')) {
-    res.sendFile(...args, err => err && next())
+    if (fallback instanceof Function) {
+      fallback(req, res, next)
```

Comment:
I feel like you should perhaps create a separate middleware that only has this conditional (HTML & GET) and leaves out further calls. Having a callback vs `sendFile` branching within this middleware, that is meant to serve only files, seems like scope creep. In return, this middleware could be refactored to use said conditional expression middleware instead of the current hard-coded `if` statement.